packages/flutter/lib/src/semantics/semantics_event.dart

```diff
@@ -86,11 +100,18 @@ class AnnounceSemanticsEvent extends SemanticsEvent {
   /// This property must not be null.
   final TextDirection textDirection;
 
+  /// Determines whether the announcement should interrupt any existing announcement, or queue after it.
+  ///
+  /// On the web this option uses the aria-live level to set the assertiveness of the announcement.
+  /// On iOS, Android, Windows, Linux, macOS, and Fuchsia this option currently has no effect.
+  final Assertiveness assertivenessSetting;
```

Comment:
nit: drop the "Setting" suffix from the name to make it shorter. In a way, all these properties are "settings".